airbyte-integrations/connectors/source-nasa-apod/source_nasa_apod/spec.yaml

```diff
@@ -0,0 +1,39 @@
+documentationUrl: https://docs.airbyte.io/integrations/sources/nasa-apod
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: NASA APOD spec
+  type: object
+  required:
+    - api_key
+  properties:
+    api_key:
+      type: string
+      description: API access key used to retrieve data from the NASA APOD API.
+      airbyte_secret: true
+    date:
```

Comment:
```suggestion
    start_date:
```

Reply:
Why you have a Date and Start date parameter?

Reply:
You can remove this parameter and use only start date.

Reply:
The API accepts both parameters, when supplied `start_date` the API responds with all images in the range [`start_date`, today]. When we supply `date`, then the API only responds with the image corresponding to the date provided.

Reply:
Conceptually connectors will try to retrieve all records from a starting point. Please remove the date parameter even if it's possible to use with the API.